inc/Engine/Optimization/RUCSS/Admin/Settings.php

```diff
@@ -513,23 +513,80 @@ public function display_error_notice() {
 			return;
 		}
 
+		$boxes = get_user_meta( get_current_user_id(), 'rocket_boxes', true );
+
+		if ( in_array( 'rucss_error_notice', (array) $boxes, true ) ) {
+			return;
+		}
+
 		$main_message = __( "It seems a security plugin or the server's firewall prevents WP Rocket from accessing the Remove Unused CSS generator. The following IP address 135.125.83.227 should be allowlisted:", 'rocket' );
 
-		$reason_1_message = __( 'in the security plugin, if you are using one', 'rocket' );
-		$reason_2_message = __( "the server's firewall - your host can help you with this", 'rocket' );
+		$reason_1_message = __( 'In the security plugin, if you are using one', 'rocket' );
+		$reason_2_message = __( "In the server's firewall - your host can help you with this", 'rocket' );
 
 		$message = sprintf(
 		// translators: %1$s = plugin name.
-			"%1\$s: <p>$main_message</p><ul><li>$reason_1_message</li><li>$reason_2_message</li></ul>",
+			"%1\$s: $main_message<ul><li>$reason_1_message</li><li>$reason_2_message</li></ul>",
```

Comment:
You should use https://developer.wordpress.org/reference/functions/wp_sprintf_l/ instead here

Reply:
@Tabrisrp this function supports HTML list layout?